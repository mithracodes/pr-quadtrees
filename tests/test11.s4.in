144.968 -37.797 144.977 -37.79
144.9678 -37.79741 144.97202 -37.79382
144.973 -37.795 144.976 -37.792
