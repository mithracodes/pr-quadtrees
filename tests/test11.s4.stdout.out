144.968 -37.797 144.977 -37.79 --> SW SW SE NE SE
144.9678 -37.79741 144.97202 -37.79382 --> SW SW SE
144.973 -37.795 144.976 -37.792 --> NE SE
